beast/physicsmodel/creategrid.py

```diff
@@ -173,8 +174,10 @@ def gen_spectral_grid_from_stellib(osl, oiso, ages=(1e7,), masses=(3,),
 
     # some constants
     kdata = 0
-    rsun = ezunits.unit['Rsun'].to('m').magnitude  # 6.955e8 m
-
+    rsun = 6.955e8  # in meters
```

Comment:
I agree.  I did not do this yet as I wanted the regression testing to not key off of small changes in the solar radius.  Will make an issue so that we make the change to astropy constants for the whole BEAST including here at some point.